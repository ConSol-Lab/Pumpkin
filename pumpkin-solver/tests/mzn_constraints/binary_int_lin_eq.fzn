var 1..3: x1 :: output_var;
var 1..3: x2 :: output_var;

constraint int_lin_eq([-1, 1], [x1, x2], 1);

solve satisfy;

