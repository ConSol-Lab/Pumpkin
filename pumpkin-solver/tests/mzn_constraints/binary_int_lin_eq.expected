x1 = 1;
x2 = 2;
----------
x1 = 2;
x2 = 3;
----------
==========
